src/client/templates/core.js.mustache

```diff
@@ -0,0 +1,11 @@
+(function () {
```

Comment:
We need better comment which describes why we need this approach. Also, let's move this files from `templates` dir  to the appropriate module dirs and rename it to `wrapped-index.js.mustache`.